src/defs/metrics.d.ts

```diff
@@ -150,10 +164,26 @@ declare module "tsdDef" {
          *
          * @param key The name of the attribute.
          * @param value The value of the attribute.
+         * @deprecated Use <code>addAnnotation</code> instead, to conform with other metrics clients.
```

Comment:
Yeah we're definitely not backwards compatible. You're saying I should just remove this, instead of deprecating? Will do